typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -415,6 +415,55 @@ export const gnosis: ChainMetadata = {
   ],
 };
 
+export const inevm: ChainMetadata = {
+  blockExplorers: [
+    {
+      apiUrl: 'https://inevm.calderaexplorer.xyz/api',
+      family: ExplorerFamily.Blockscout,
+      name: 'Caldera inEVM Explorer',
+      url: 'https://inevm.calderaexplorer.xyz/',
+    },
+  ],
+  blocks: {
+    confirmations: 1,
+    estimateBlockTime: 3,
+    reorgPeriod: 0,
+  },
+  chainId: 2525,
+  domainId: 2525,
+  displayName: 'Injective EVM',
+  displayNameShort: 'inEVM',
+  name: Chains.inevm,
+  nativeToken: {
+    decimals: 18,
+    name: 'Injective',
+    symbol: 'INJ',
+  },
+  protocol: ProtocolType.Ethereum,
+  rpcUrls: [{ http: 'https://inevm.calderachain.xyz/http' }],
+};
+
+export const injective: ChainMetadata = {
+  blockExplorers: [],
+  blocks: {
+    confirmations: 1,
+    estimateBlockTime: 3,
+    reorgPeriod: 1,
+  },
+  chainId: 'injective-1',
+  domainId: 6909546,
+  displayName: 'Injective',
+  displayNameShort: 'Injective',
+  name: Chains.injective,
+  nativeToken: {
+    decimals: 6,
+    name: 'Injective',
+    symbol: 'INJ',
+  },
+  protocol: ProtocolType.Cosmos,
+  rpcUrls: [{ http: 'https://sentry.tm.injective.network:443' }],
```

Comment:
Chains chains also need a `restURLs` field for the warp UI to work right (see neutron)